### PR TITLE
Clarifying roles required to setup Azure DevOps

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Updating the Azure DevOps guide to include which personas are needed for each section

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/cloud/git/setup-azure.md [hunk 1]
```diff
@@ -1,49 +1,53 @@
...
+1. [Register an Entra ID app](#register-an-entra-id-app).
 2. [Add permissions to your new app](#add-permissions-to-your-new-app).
 3. [Add another redirect URI](#add-another-redirect-uri).
 4. [Connect Azure DevOps to your new app](#connect-azure-devops-to-your-new-app).
-5. [Add your Azure AD app to dbt Cloud](#add-your-azure-ad-app-to-dbt-cloud).
+5. [Add your Entra ID app to dbt Cloud](#add-your-azure-ad-app-to-dbt-cloud).
 
-Once the Azure AD app is added to dbt Cloud, an account admin must also connect a service user via OAuth, which will be used to power headless actions in dbt Cloud such as deployment runs and CI.
-1. [Connecting a Service User](#connecting-a-service-user).
+Once the Entra ID app is added to dbt Cloud, an account admin must also [connect a service user]((#connecting-a-service-user)) via OAuth, which will be used to power headless actions in dbt Cloud such as deployment runs and CI.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/git/setup-azure.md]:
```suggestion
Once the Entra ID app is added to dbt Cloud, an account admin must also [connect a service user](#connecting-a-service-user) via OAuth, which will be used to power headless actions in dbt Cloud such as deployment runs and CI.
```

File: website/docs/docs/cloud/git/setup-azure.md [hunk 2]
```diff
@@ -1,49 +1,53 @@
...
+The following personas are required to complete the steps on this page:
+- Entra ID admin
+- Azure DevOps admin
+- dbt Cloud account admin
 
-## Register an Azure AD app
+## Register an Entra ID app
 
-1. Sign into your Azure portal and click **Azure Active Directory** under Azure services.
+An Entra ID account admin needs to perform the following steps:
```
Comment (matthewshaver) [inline website/docs/docs/cloud/git/setup-azure.md]:
```suggestion
An Entra ID admin needs to perform the following steps:
```

File: website/docs/docs/cloud/git/setup-azure.md [hunk 3]
```diff
@@ -55,49 +59,63 @@ Provide your new app access to Azure DevOps:
...
+1. Navigate to your Entra ID application.
+2. Select **Certificates and Secrets** from the left navigation panel.
+3. Select **Client secrets** and click **New client secret**
+4. Give the secret a description and select the expiration time. Click **Add**.
+5. Copy the **Value** field and securely share it with the dbt Cloud account admin who will complete the setup. 
 
 ## Connect Azure DevOps to your new app
-If you have already connected your Azure DevOps account to Active Directory, then you can proceed to [Connecting a service user](#connecting-a-service-user). However, if you're just getting set up, connect Azure DevOps to the Active Directory App you just created:
+
+An Azure admin will need one of the following permissions in both the Entra ID and Azure DevOps environments:
```
Comment (b-per) [inline website/docs/docs/cloud/git/setup-azure.md:85]: Is "Azure admin" another persona? Or is ti the same as one of the others?

If it is not, can we add it at the top in the list of personas a(and maybe mention what sort of admin and/or access it requires)

File: website/docs/docs/cloud/git/setup-azure.md [hunk 4]
```diff
@@ -343,7 +359,7 @@ You must connect your service user before setting up a dbt Cloud project, as the
 To connect the service user:
 1. An admin must first be signed into the service user's Azure DevOps account.
```
Comment (b-per) [inline website/docs/docs/cloud/git/setup-azure.md]: Which Admin is it in that case?

**Comment by b-per** [comment]
Thanks @matthewshaver !
I have added 2 comments

File: website/docs/docs/cloud/git/setup-azure.md (hunk 3 above)
Comment (matthewshaver) [reply website/docs/docs/cloud/git/setup-azure.md:85]: @b-per Sure, I'll add it up top.

**Comment by matthewshaver** [comment]
Thanks @b-per I addressed the comments, hopefully to your satisfaction 😄

**Review by b-per — APPROVED** [review]
Thanks @matthewshaver , this is great 🙌 
I am sure it will help quite a few folks to navigate the many steps involved

File: website/docs/docs/cloud/git/setup-azure.md [hunk 5]
```diff
@@ -340,11 +357,11 @@ az devops security permission update --organization https://dev.azure.com/<org_n
 
 You must connect your service user before setting up a dbt Cloud project, as the service user's permissions determine which projects dbt Cloud can import.
 
-To connect the service user:
-1. An admin must first be signed into the service user's Azure DevOps account.
-2. The admin should click **Link Azure Service User** in dbt Cloud.
-3. The admin will be directed to Azure DevOps and must accept the Azure AD app's permissions.
-4. Finally, the admin will be redirected to dbt Cloud, and the service user will be connected.
+A dbt Cloud account admin with access to the service users Azure DevOps account must complete the following to connect the service user:
```
Comment (matthewshaver) [inline website/docs/docs/cloud/git/setup-azure.md]:
```suggestion
A dbt Cloud account admin with access to the service user's Azure DevOps account must complete the following to connect the service user:
```

File: website/docs/docs/cloud/git/setup-azure.md [hunk 6]
```diff
@@ -340,11 +357,11 @@ az devops security permission update --organization https://dev.azure.com/<org_n
 
 You must connect your service user before setting up a dbt Cloud project, as the service user's permissions determine which projects dbt Cloud can import.
 
-To connect the service user:
-1. An admin must first be signed into the service user's Azure DevOps account.
-2. The admin should click **Link Azure Service User** in dbt Cloud.
-3. The admin will be directed to Azure DevOps and must accept the Azure AD app's permissions.
-4. Finally, the admin will be redirected to dbt Cloud, and the service user will be connected.
+A dbt Cloud account admin with access to the service user's Azure DevOps account must complete the following to connect the service user:
+1. Sign into the service user's Azure DevOps account.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/git/setup-azure.md]:
```suggestion
1. Sign in to the service user's Azure DevOps account.
```

**Pull request merged**